### PR TITLE
VPN post-download pages for Windows & Mac

### PR DESCRIPTION
**Pull request by reemhamz**
## One-line summary
When the user chooses to download Mozilla VPN from `/vpn/download` (created [HERE](https://github.com/mozilla/bedrock/pull/12980)) for either Windows or Mac, they will be directed to the respective pages of the platform they chose, and then the download should start automatically. The logic here is similar to the [`/thanks` page](https://www.mozilla.org/en-US/firefox/download/thanks/) for Firefox.

- [FAQ copy](https://docs.google.com/document/d/18XdO2o0aYW22lzBcqrVe_O-mbqAieL0e3Bz0QYYOMQY/edit#)
- [Figma file](https://www.figma.com/file/FjeA0eZw4GIxBAlQfz06c1/VPN-Download-page-%255B%252B-Archived-Landing-page%255D?node-id=2608-7141&t=t2u25N2eTn0JS30T-0&fuid=1005135947042770759)

## Issue / Bugzilla link
#11875 

## Testing

Make sure to test on Browserstack for the alternative platform to check automatic download

### Local testing

- [ ] http://localhost:8000/en-US/products/vpn/download/mac/thanks
- [ ] http://localhost:8000/en-US/products/vpn/download/windows/thanks

### Demo servers:
- [ ] https://www-demo4.allizom.org/en-US/products/vpn/download/mac/thanks/
- [ ] https://www-demo4.allizom.org/en-US/products/vpn/download/windows/thanks/

### CODE REVIEW

File: l10n/en/products/vpn/platform-post-download.ftl [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+vpn-download-faq-experience = If your goal is to experience the internet as if you were in some other part of the world, you can choose a server location there and all websites you visit will display as if you were there.
+vpn-download-faq-protect = If you want to protect yourself even more, you can activate protections against Ads, Ad Trackers and Malware by going to Settings > Privacy features. Please note that this may cause some websites to break or not load, so you might need to temporarily disable these protections if you bump into this issue.
+vpn-download-faq-lastly = Lastly, on supported devices, you can make the VPN even smarter by excluding certain apps (ex. Netflix) from the VPN.
+
+vpn-download-faq-need-more = Need a little more help?
+
+# Variables
+#   $support (url) - link to https://support.mozilla.org/en-US/products/firefox-private-network-vpn
+vpn-download-faq-support = You can contact our support team right from the { -brand-name-mozilla-vpn } Application by clicking Settings > Get Help > Contact Support. You can also contact us <a href="{ $support }">on our support page</a>.
```
Comment (reemhamz) [inline l10n/en/products/vpn/platform-post-download.ftl:65]: I changed the anchor string from `here` to `on our support page` for better a11y

File: l10n/en/products/vpn/platform-post-download.ftl [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+vpn-turn-on-vpn = 4. Turn on VPN
+vpn-pick-a-recommended = Pick a recommended location for a fast, stable connection or choose your desired location manually, then switch the toggle ON
+
+vpn-download-faq-need-help = Need help getting started?
+vpn-download-faq-here-are-some = Here are some of the most common questions about setting up { -brand-name-mozilla-vpn }
+vpn-download-faq-why-use = Why should I use { -brand-name-mozilla-vpn }
+
+# Variables:
+#   $faq (url) - link to https://www.mozilla.org/en-CA/products/vpn/more/what-is-a-vpn/
+vpn-download-faq-as-more-daily = As more of daily life takes place through the internet, online privacy and security become even more important. A { -brand-name-vpn }, <a href="{ $faq}">Virtual Private Network</a>, creates an encrypted tunnel between your devices and the internet, ensuring your connection remains secure and private while protecting you in two important ways:
```
Comment (alexgibson) [inline l10n/en/products/vpn/platform-post-download.ftl:35]: Suggestion (non blocking): it's a little hard to figure out which page each string relates to here. Might it make sense to organize the strings into groups to make it a little clearer to read? e.g.

```
## Window download page

... INSERT_WINDOWS_PAGE_STRINGS ...

## Mac Download page

... INSERT_MAC_PAGE_PAGE_STRINGS ...

## Shared strings

... INSERT_SHARED_STRINGS ...

## FAQ Strings

... INSERT_FAQ_STRINGS ...
```

File: l10n/en/products/vpn/platform-post-download.ftl [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+vpn-enter-your-email = Enter your email address to log in or sign up and get started
+vpn-turn-on-vpn = 4. Turn on VPN
+vpn-pick-a-recommended = Pick a recommended location for a fast, stable connection or choose your desired location manually, then switch the toggle ON
+
+vpn-download-faq-need-help = Need help getting started?
+vpn-download-faq-here-are-some = Here are some of the most common questions about setting up { -brand-name-mozilla-vpn }
+vpn-download-faq-why-use = Why should I use { -brand-name-mozilla-vpn }
+
+# Variables:
+#   $faq (url) - link to https://www.mozilla.org/en-CA/products/vpn/more/what-is-a-vpn/
```
Comment (alexgibson) [inline l10n/en/products/vpn/platform-post-download.ftl:34]: Nit remove locale
```suggestion
#   $faq (url) - link to https://www.mozilla.org/products/vpn/more/what-is-a-vpn/
```

File: l10n/en/products/vpn/platform-post-download.ftl [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+vpn-download-faq-as-more-daily = As more of daily life takes place through the internet, online privacy and security become even more important. A { -brand-name-vpn }, <a href="{ $faq}">Virtual Private Network</a>, creates an encrypted tunnel between your devices and the internet, ensuring your connection remains secure and private while protecting you in two important ways:
+
+# Variables:
+#   $ip (url) - link to https://www.mozilla.org/en-CA/products/vpn/more/what-is-an-ip-address/
```
Comment (alexgibson) [inline l10n/en/products/vpn/platform-post-download.ftl:38]: Nit: remove locale
```suggestion
#   $ip (url) - link to https://www.mozilla.org/products/vpn/more/what-is-an-ip-address/
```

File: l10n/en/products/vpn/platform-post-download.ftl [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+vpn-download-faq-concealing-ip = Concealing your true <a href="{ $ip }">IP address</a>: this helps protect your identity and obscures your location from the websites you visit.
+vpn-download-faq-encrypting = Encrypting your internet traffic: so no one on the wire can spy on your online activity.
+
+vpn-download-faq-working = How do I know the { -brand-name-vpn } is working?
+
+# Variables:
+#   $connected (url) - link to https://support.mozilla.org/en-US/kb/how-can-i-tell-if-mozilla-vpn-connected
```
Comment (alexgibson) [inline l10n/en/products/vpn/platform-post-download.ftl:45]: Nit: remove locale
```suggestion
#   $connected (url) - link to https://support.mozilla.org/kb/how-can-i-tell-if-mozilla-vpn-connected
```

File: l10n/en/products/vpn/platform-post-download.ftl [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+vpn-download-faq-visual-indicators = { -brand-name-mozilla-vpn } displays visual indicators of its current status both in the toolbar and the application’s home screen, making it easy to know whether your online activity is protected or not. These indicators allow you to confirm when your navigation is private and secure. Additionally, while connected, you can visit <a href="https://monitor.firefox.com/">https://monitor.firefox.com/</a> to confirm if your IP address is masked. For more details, please see <a href="{ $connected }">How can I tell if { -brand-name-mozilla-vpn } is connected?</a>
+
+vpn-download-faq-add-device = How do I add another device?
+
+# Variables:
+#   $subscription - link to https://support.mozilla.org/en-US/kb/how-add-devices-your-mozilla-vpn-subscription
```
Comment (alexgibson) [inline l10n/en/products/vpn/platform-post-download.ftl:51]: Nit: remove locale
```suggestion
#   $subscription - link to https://support.mozilla.org/kb/how-add-devices-your-mozilla-vpn-subscription
```

File: l10n/en/products/vpn/platform-post-download.ftl [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+vpn-download-faq-adding-another = Adding another device is as simple as downloading and installing the { -brand-name-mozilla-vpn } software and then signing into your existing { -brand-name-firefox } account on the new device. For more details, please see <a href="{ $subscription }">How to add devices to your { -brand-name-mozilla-vpn } subscription </a>
+
+
+vpn-download-faq-best-practices = What are some { -brand-name-vpn } best practices?
```
Comment (alexgibson) [inline l10n/en/products/vpn/platform-post-download.ftl:55]: Same here, perhaps the brand name is not needed?

File: media/js/products/vpn/thanks.js [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+     */
+    DownloadThanks.shouldAutoDownload = function (platform) {
+        var supportedPlatforms = ['windows', 'osx'];
+
+        if (supportedPlatforms.indexOf(platform) !== -1) {
+            return true;
+        }
+
+        return false;
+    };
```
Comment (alexgibson) [inline media/js/products/vpn/thanks.js:30]: As above, I think we can remove this

File: media/js/products/vpn/thanks.js [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+                link = document.getElementById(prefix + 'mac');
+                break;
+        }
+
+        if (link && link.href) {
+            url = link.href;
+        }
+
+        return url;
+    };
```
Comment (alexgibson) [inline media/js/products/vpn/thanks.js:56]: Issue (blocking) I'm not sure we need this either? Right now if I'm on mac and click to try and download a windows installer, I get nothing since this is doing a platform based check.

I don't think we need to keep this as copy over from how we do Firefox downloads, since here we have two separate download pages and not a single page that handles all platforms.

We can probably remove this file entirely, and just have a separate JS bundle for both the Windows and Mac download pages, that triggers the correct file download automatically? This feels like it might be easier to maintain than a shared file?

Let me know if you want to talk through this if something I've said doesn't make sense 🙂

File: media/static-bundles.json [hunk 10]
```diff
@@ -1940,6 +1947,14 @@
       ],
       "name": "mozilla-vpn-invite"
     },
+    {
+      "files": [
+        "protocol/js/protocol-details.js",
```
Comment (alexgibson) [inline media/static-bundles.json:1952]: Nit: I don't think you need to add `protocol-details.js` here as it's already in our common bundle?

File: media/js/products/vpn/thanks-init.js [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+
+        if (downloadURL) {
+            // Pull download link from the download button and add to the 'Try downloading again' link.
+            // Make sure the 'Try downloading again' link is well formatted! (issue 9615)
+            if (directDownloadLink && directDownloadLink.href) {
+                directDownloadLink.href = downloadURL;
+            }
+
+            // Start the platform-detected download a second after DOM ready event.
+            // We don't rely on the window load event as we have third-party tracking pixels.
```
Comment (alexgibson) [inline media/js/products/vpn/thanks-init.js:28]: Nit: there's no tracking pixel on this line so I think you can remove this comment

**Comment by alexgibson** [comment]
OH - I just thought of another potential blocker: we need to make sure people can't download VPN in countries where we're not legally allowed to let them. On both of these post-download pages, we need to stop the download from happening in those countries (and probably display a message saying why, like we do on the main `/download/` page).

**Comment by alexgibson** [comment]
I forgot to ask earlier: do we need to make sure this has been pushed to a demo and shared with stakeholders for feedback before merging?

File: bedrock/products/templates/products/vpn/windows-download.html [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "products/vpn/download.html" %}
+
```
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/windows-download.html:13]: Question (potentially blocking): As these pages auto-download a file on load, we probably want to exclude them from search engines? Worth confirming with Adria here?

```
{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
```

File: bedrock/products/templates/products/vpn/mac-download.html [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+{% endblock %}
+
+{% block content %}
+  <div class="vpn-platform-download">
+    <header class="platform-download-header">
+      <div class="mzp-l-content platform-download-header-wrapper">
+        <div class="vpn-logo">
+          <img src="{{ static('img/products/vpn/download/vpn-dark-logo.svg') }}" alt="">
+        </div>
+        <h1>{{ ftl('vpn-mac-downloading-mozilla-vpn') }}...</h1>
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/products/templates/products/vpn/mac-download.html:23]: ha! I didn't know that was a stand alone character. TIL

File: l10n/en/products/vpn/platform-post-download.ftl (hunk 2 above)
[the remark this answers is not in the document]
Comment (reemhamz) [reply l10n/en/products/vpn/platform-post-download.ftl:35]: Fair point. I was hesitant to use the `brand-name-vpn` variable, but since I noticed that it existed, I assumed it could be used in this circumstance, but I'm happy to remove it.

File: media/js/products/vpn/thanks-init.js [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+    'use strict';
+
+    var directDownloadLink = document.getElementById('vpn-download-link');
+    var downloadURL;
+
+    // Only auto-start the download if a supported platform is detected.
+    if (
+        Mozilla.DownloadThanks.shouldAutoDownload(window.site.platform) &&
+        typeof Mozilla.Utils !== 'undefined'
+    ) {
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply media/js/products/vpn/thanks-init.js:17]: But the download packages for each platform are different? Curious why we're allowing a Windows download on a Mac and vice versa? Is it in case someone is masking their platform?

File: bedrock/products/views.py [hunk 15]
```diff
@@ -95,6 +95,28 @@ def vpn_download_page(request):
     return l10n_utils.render(request, template_name, context, ftl_files=ftl_files)
 
 
+@require_safe
+def vpn_mac_download_page(request):
+    template_name = "products/vpn/mac-download.html"
+    ftl_files = ["products/vpn/platform-post-download", "products/vpn/shared"]
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/products/views.py:101]: Would adding this `reference` be correct? :
```[[paths]]
    reference = "en/products/vpn/download/*.ftl"
    l10n = "{locale}/products/vpn/platform-post-download.ftl"
```
Asking in case, because there is `reference = "en/products/vpn/download.ftl"` for the download page

File: bedrock/products/templates/products/vpn/windows-download.html (hunk 12 above)
Comment (reemhamz) [reply bedrock/products/templates/products/vpn/windows-download.html:13]: I'll ask Adria :)

Comment (reemhamz) [reply bedrock/products/templates/products/vpn/windows-download.html:13]: She agreed that we should noindex the page, and she also mentioned to nofollow the links to the post-download pages, so I'll be adding `rel="nofollow"` in the `/vpn/download` page

File: media/css/products/vpn/platform-download.scss [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+        }
+    }
+
+    p {
+        @include text-body-xl;
+    }
+}
+
+.mzp-c-picto {
+    text-align: center;
```
Comment (reemhamz) [inline media/css/products/vpn/platform-download.scss:37]: Aligned text center to match the images

File: media/img/products/vpn/download/mac-installer.png
Comment (reemhamz) [inline media/img/products/vpn/download/mac-installer.png:1]: These post-download page instruction images are larger than we'd like, but when we had them as SVGs, we were getting blurry images on Webkit browsers + iOS devices because of a known issue with how Webkit renders SVGs with drop-shadows. 
Though larger, we needed images that were clear, especially on mobile browsers, because the images help with user instruction for VPN installation

File: bedrock/mozorg/redirects.py [hunk 17]
```diff
@@ -16,8 +16,6 @@ def decider(request, **kwargs):
 redirectpatterns = (
     # bug 755826, 1222348, 1416798
     redirect(r"^zh-CN/?$", "https://www.firefox.com.cn/", locale_prefix=False, query={"utm_medium": "referral", "utm_source": "mozilla.org"}),
-    # bug 874913, 681572
-    redirect(r"^(products/)?download\.html", "firefox.new", query=""),
```
Comment (alexgibson) [inline bedrock/mozorg/redirects.py:20]: I think this is unrelated to your redirect issues and you can / should restore this.

File: tests/redirects/map_globalconf.py [hunk 18]
```diff
@@ -32,8 +32,8 @@
         # bug 755826, 1222348, 1416798
         url_test("/zh-CN/", "https://www.firefox.com.cn/", query={"utm_medium": "referral", "utm_source": "mozilla.org"}),
         # bug 874913
-        url_test("/en-US/{,products/}download.html{,?stuff=whatnot}", "/en-US/firefox/new/"),
-        url_test("/{,products/}download.html{,?stuff=whatnot}", "/firefox/new/"),
+        # url_test("/en-US/{,products/}download.html{,?stuff=whatnot}", "/en-US/firefox/new/"),
+        # url_test("/{,products/}download.html{,?stuff=whatnot}", "/firefox/new/"),
```
Comment (alexgibson) [inline tests/redirects/map_globalconf.py:36]: These redirect tests should be left in, they do not seem to be related to your issue from my testing?

File: bedrock/base/redirects.py [hunk 19]
```diff
@@ -0,0 +1,7 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from bedrock.redirects.util import redirect
+
+redirectpatterns = (redirect(r"^vpn/download/?$", "products.vpn.download"),)
```
Comment (alexgibson) [inline bedrock/base/redirects.py:7]: Suggestion: add a comment here to explain why this redirect is here, and not in `products/redirects.py`?

File: bedrock/products/templates/products/vpn/mac-download.html [hunk 20]
```diff
@@ -0,0 +1,110 @@
...
+    </div>
+    {% include 'products/vpn/includes/download-faq.html' %}
+  {% endif %}
+  <footer class="vpn-footer">
+      {% include 'products/vpn/includes/footer-legal.html' %}
+  </footer>
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle('vpn_platform_download') }}
```
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/mac-download.html:109]: Issue: the reason why you get redirected to a 404 when we block downloads is because this JS file still tries to run (but finds no download and redirects you to `undefined`. Suggest adding a conditional round the bundle:
```suggestion
  {% if not block_download %}
    {{ js_bundle('vpn_platform_download') }}
  {% endif %
```

File: bedrock/products/templates/products/vpn/windows-download.html [hunk 21]
```diff
@@ -0,0 +1,110 @@
...
+  </div>
+  {% include 'products/vpn/includes/download-faq.html' %}
+  {% endif %}
+  <footer class="vpn-footer">
+      {% include 'products/vpn/includes/footer-legal.html' %}
+  </footer>
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle('vpn_platform_download') }}
```
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/windows-download.html:109]:
```suggestion
  {% if not block_download %}
    {{ js_bundle('vpn_platform_download') }}
  {% endif %}
```

File: l10n/en/products/vpn/platform-post-download.ftl [hunk 22]
```diff
@@ -0,0 +1,73 @@
...
+vpn-mac-follow-the-steps = Follow the steps outlined in the { -brand-name-mozilla-vpn } installer to start the app
+
+## Shared strings
+
+# Variables:
+#   $url (url) - link for either Windows or Mac download, depending on user platform
+#   $id (string) - unique ID for the anchor tag
+# The <br> tag is a line break for the HTML text
+vpn-download-unable-to-download = Unable to download { -brand-name-mozilla-vpn }
+vpn-download-not-in-country = Sorry, { -brand-name-mozilla-vpn } is not available in your country .
```
Comment (alexgibson) [inline l10n/en/products/vpn/platform-post-download.ftl:29]: Nit: remove trailing space
```suggestion
vpn-download-not-in-country = Sorry, { -brand-name-mozilla-vpn } is not available in your country.
```

File: l10n/en/products/vpn/platform-post-download.ftl [hunk 23]
```diff
@@ -0,0 +1,73 @@
...
+
+## FAQ Strings
+
+vpn-download-faq-need-help = Need help getting started?
+vpn-download-faq-here-are-some = Here are some of the most common questions about setting up { -brand-name-mozilla-vpn }
+vpn-download-faq-why-use = Why should I use { -brand-name-mozilla-vpn }
+
+# Variables:
+#   $faq (url) - link to https://www.mozilla.org/products/vpn/more/what-is-a-vpn/
+vpn-download-faq-as-more-daily = As more of daily life takes place through the internet, online privacy and security become even more important. A VPN, <a href="{ $faq}">Virtual Private Network</a>, creates an encrypted tunnel between your devices and the internet, ensuring your connection remains secure and private while protecting you in two important ways:
```
Comment (alexgibson) [inline l10n/en/products/vpn/platform-post-download.ftl:47]: Nit: add space
```suggestion
vpn-download-faq-as-more-daily = As more of daily life takes place through the internet, online privacy and security become even more important. A VPN, <a href="{ $faq }">Virtual Private Network</a>, creates an encrypted tunnel between your devices and the internet, ensuring your connection remains secure and private while protecting you in two important ways:
```

File: media/js/products/vpn/thanks.js [hunk 24]
```diff
@@ -0,0 +1,28 @@
...
+    if (document.getElementById(prefix + 'win') !== null) {
+        link = document.getElementById(prefix + 'win').href;
+    } else if (document.getElementById(prefix + 'mac') !== null) {
+        link = document.getElementById(prefix + 'mac').href;
+    }
+
+    // Trigger auto-download based on platform page
+    Mozilla.Utils.onDocumentReady(function () {
+        setTimeout(function () {
+            window.location.href = link;
```
Comment (alexgibson) [inline media/js/products/vpn/thanks.js:22]: Suggestion: let's only do a redirect if a link is found (which can avoid errors like redirecting to `undefined`?
```suggestion
            if (link) {
                window.location.href = link;
            }
```

File: media/js/products/vpn/thanks.js [hunk 25]
```diff
@@ -0,0 +1,28 @@
...
+    'use strict';
+    var prefix = 'vpn-download-link-';
+    var link;
+
+    // Check for download link on Windows or Mac download pages
+    if (document.getElementById(prefix + 'win') !== null) {
+        link = document.getElementById(prefix + 'win').href;
+    } else if (document.getElementById(prefix + 'mac') !== null) {
+        link = document.getElementById(prefix + 'mac').href;
+    }
```
Comment (alexgibson) [inline media/js/products/vpn/thanks.js:17]: Suggestion: can cut down on the number of DOM queries here by using a variable?

```javascript
    var win = document.getElementById(prefix + 'win');
    var mac = document.getElementById(prefix + 'mac');

    // Check for download link on Windows or Mac download pages
    if (win) {
        link = win.href;
    } else if (mac) {
        link = mac.href;
    }
```

File: bedrock/products/templates/products/vpn/windows-download.html [hunk 26]
```diff
@@ -0,0 +1,110 @@
...
+        <p>{{ ftl('vpn-your-download-should-start', url=windows_download_url, id='vpn-download-link-win') }}</p>
+      </div>
+    </header>
+    <section class="vpn-download-instructions windows mzp-l-content">
+      <ol class="mzp-l-columns mzp-t-columns-four">
+        {% call picto(
+          base_el='li',
+          title=ftl('vpn-open-the-file'),
+          image=resp_img(
+            url='img/products/vpn/download/vpn-file-windows.png',
```
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/windows-download.html:46]: Suggestion: it would be nice to add a responsive image for these so we're not serving larger images than are needed?

https://bedrock.readthedocs.io/en/latest/coding.html#resp-img

**Review by alexgibson — CHANGES_REQUESTED** [review]
The redirects seems to be working for me (my guess is you had an old / bad redirect cached in your browser, which is the thing that's been tripping you up?)

Also left a couple of other suggestions, and a fix for the `undefined` download issue 👍

File: media/img/products/vpn/download/mac-installer.png
Comment (alexgibson) [reply media/img/products/vpn/download/mac-installer.png:1]: It would be nice to add a couple of different sizes for these images using `resp_img`, so the browser can work out the best size to serve? Happy to walk through / pair on doing this if it's a helpful way to learn :)

File: media/js/products/vpn/thanks.js (hunk 24 above)
Comment (reemhamz) [reply media/js/products/vpn/thanks.js:22]: 🧠 🌌

File: bedrock/products/templates/products/vpn/mac-download.html [hunk 27]
```diff
@@ -0,0 +1,148 @@
...
+              url='img/products/vpn/download/vpn-file-mac-700.png',
+              srcset={
+              "img/products/vpn/download/vpn-file-mac-960.png": "960w",
+              "img/products/vpn/download/vpn-file-mac-700.png": "700w",
+              "img/products/vpn/download/vpn-file-mac-500.png": "500w",
+            },
+            sizes={
+              '(min-width: 768px)': '760px',
+              'default': '250px'
+            },
```
Comment (alexgibson) [inline bedrock/products/templates/products/vpn/mac-download.html:55]: Sugggestion: worth adding a couple of different breakpoints for when the picto images split between tow and four rows?
```suggestion
            sizes={
              '(min-width: 1024px)': '25vw',
              '(min-width: 768px)': '50vw',
              'default': '250px'
            },
```

File: bedrock/products/redirects.py [hunk 28]
```diff
@@ -5,6 +5,8 @@
 from bedrock.redirects.util import redirect
 
 redirectpatterns = (
+    # Issue 10335
+    redirect(r"^vpn/?$", "products.vpn.landing"),
```
Comment (alexgibson) [inline bedrock/products/redirects.py:9]: Suggestion: worth adding a test for this existing redirect?

**Review by alexgibson — CHANGES_REQUESTED** [review]
A couple of small last suggestions, but otherwise I think this is ready to merge. Stellar work here Reem! 💯 r+wc

File: bedrock/products/redirects.py (hunk 28 above)
Comment (reemhamz) [reply bedrock/products/redirects.py:9]: It already exists here from a previous time: https://github.com/mozilla/bedrock/blob/main/tests/redirects/map_globalconf.py#L1176
I added it in this PR some time ago: https://github.com/mozilla/bedrock/pull/10338/files#diff-f0a4473a733687326b397105e2d566e6c377bf59fc212f3032e6e4a2882d6551R1393

**Review by alexgibson — APPROVED** [review]
r+ 🍡
